src/streams.c

```diff
@@ -308,7 +308,7 @@ Obj READ_AS_FUNC ( void )
 }
 
 
-static void READ_TEST_OR_LOOP(void)
+static void READ_TEST_OR_LOOP(Obj context)
```

Comment:
Huh -- `READ_TEST_OR_LOOP` does not use the `context` argument at all. Was some code lost here or what?